contracts/Oath.sol

```diff
@@ -4,11 +4,11 @@ pragma solidity 0.8.9;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
-// The token
+// The token (test version)
 contract Oath is ERC20 {
     constructor(string memory name, string memory symbol) ERC20(name, symbol) {}
 
-    // TODO tess3rac7 anyone can mint?
+    // anyone can mint(!) since it's test version
```

Comment:
Just curious, but wouldn't it be safer to add a parameter or private property somewhere setting if the deployment was on testnet or not?
Not sure if that's possible or what downsides could be. The risk of something like this slipping to mainnet might be small, but the impact is not.
From a perspective as devops engineer,  it usually is desired to deploy the same thing on multiple environments, and only change the parameters (in this case it'd be something like a global-ish property that defines if the deployment is on test- or main-net) and use that in applicable functions.
But i can imagine this won't apply on the blockchain due to it's immutable nature. But curious about your thoughts

Reply:
umm... as for us, we have a separate repo for the actual Oath token. This particular instance is just a dummy for tests to go along with the Reliquary.

Re: your concern of folks cloning this and accidentally deploying this instance of Oath on mainnet as their actual Oath token, I guess we could just rename the file to TestOath or something?